src/controllers/EventsController.js

```diff
@@ -8,7 +8,7 @@ import anonymous from '../decorators/anonymous';
 import route from '../decorators/route';
 import httpVerb from '../decorators/httpVerb';
 import serverSentEvents from '../decorators/serverSentEvents';
-import logger from '../lib/logger';
+import Logger from '../lib/logger'; const logger = Logger.createModuleLogger(module);
```

Comment:
`const logger = Logger.createModuleLogger(module);` should come after all the imports